recipes/citrination-client/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "citrination-client" %}
+{% set version = "5.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e20fbbac5339153707a8976e68ce7d64f9768aace9278d7e189791abd4d7d2e4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you actually need a C compiler, it looks like this is pure Python? If you do need one this probably can't be a noarch package.
(same applies for `pypif`)

Reply:
Thanks -- good call!